### ISSUE TITLE
Can't block senders

### ISSUE THREAD

**Issue by stateid** [simple-login/Simple-Login-Android#65]
I didn't find an option to block a sender inside an alies.

**Comment by matbgn** [comment]
I have to agree. I also didn't have found how to block them on Android, analog to the so called "_Contacts_" webpage interface (see below). Is there a way to do that? Or at least a plan to integrate it?

![image](https://user-images.githubusercontent.com/13169819/166870024-8eab5213-478a-48c8-b200-27c58b60aceb.png)
